Makefile

```diff
@@ -50,3 +50,9 @@ kernel-%: ## Updates the kernel configs: e.g. make kernel-olddefconfig; make ker
 		$(MAKE) docker-kernel-prepare PLATFORM=$$platform TARGET_ARGS="--tag=$(REGISTRY)/$(USERNAME)/kernel:$(TAG)-$$arch --load"; \
 		docker run --rm -it --entrypoint=/toolchain/bin/bash -e PATH=/toolchain/bin:/bin -w /src -v $$PWD/kernel/kernel/config-$$arch:/host/.hostconfig $(REGISTRY)/$(USERNAME)/kernel:$(TAG)-$$arch -c 'cp /host/.hostconfig .config && make $* && cp .config /host/.hostconfig'; \
 	done
+
+# Utilities
+
+.PHONY: conformance
```

Comment:
can we call this stage `conform` instead of `conformance`? conformance implies the k8s conformance process to me mentally, wheras I know conform is its own project.

Reply:
should we update here only or other places too, all repos have `conformance`

Reply:
oh, well if it's already like that in other places, carry on. I was just being a pain this morning I guess :D